patterns/multisig/proxy.ts

```diff
@@ -0,0 +1,136 @@
+import { types } from "polkadot_dev/mod.ts"
+import { Proxy, Utility } from "polkadot_dev/mod.ts"
+import { Event } from "polkadot_dev/types/pallet_proxy/pallet.ts"
+import { RuntimeEvent } from "polkadot_dev/types/polkadot_runtime.ts"
+import { ArrayRune, MultiAddress, Rune, RunicArgs } from "../../mod.ts"
+import { hex, Sr25519 } from "../../util/mod.ts"
```

Comment:
These are exposed through the root `mod.ts` as well (can move `hex, Sr25519` into the preceding import item list)